grouper/api/handlers.py

```diff
@@ -95,25 +111,15 @@ def write_error(self, status_code, **kwargs):
 class Users(GraphHandler):
     def get(self, name=None):
         cutoff = int(self.get_argument("cutoff", 100))
-        include_role_users = self.get_argument("include_role_users", "no") == "yes"
```

Comment:
We use this parameter in a bunch of places internally. Will we break existing clients?